src/main/scala/com/exasol/cloudetl/kafka/KafkaConsumerProperties.scala

```diff
@@ -361,6 +369,19 @@ object KafkaConsumerProperties extends CommonProperties {
     "EXASOL_KAFKA_UDFS_CONSUMERS"
   )
 
+  /**
+   * This is the {@code auto.offset.reset} configuration setting.
+   *
+   * This controls where the consumer starts when no previous offsets have been inserted in the
+   * table or the offset stored in the table is out of range in the partition.
+   * Defaults to {@code earliest}.
+   */
+  private[kafka] final val AUTO_OFFSET_RESET: Config[String] = Config[String](
```

Comment:
Could you please add this to optional parameters in user guide?

Does using this parameter imply the duplicate imports? I think it does not work if the Exasol table already have offsets.

Reply:
See comment above, and additionally, I don't know if this even should be configurable. The importer code guards against database overload with `MAX_RECORD_PER_RUN` and `MIN_RECORD_PER_RUN` already, so maybe it would be better to just set the default to earliest to prevent users from losing records

Reply:
Alright, let us keep as default.